### ISSUE TITLE
Please use an unambiguous date format on the chocolatey.org website

### ISSUE THREAD

**Issue by hi2u** [chocolatey/chocolatey.org#735]
### What You Are Seeing?

Go to the details page for any package, e.g: https://chocolatey.org/packages/nircmd

In the sidebar on the left, it shows `mm/dd/yyyy` formatted dates, which are both:

* confusing for 95% of the world's population
* and ambiguous to everyone for 40% of every year, i.e. the first 12 days of every month. `dd/mm/yyyy` has the same problem.  

They're probably displayed elsewhere too.  So I'm suggesting a better format is used everywhere.

### What is Expected?

Any format that is unambiguous. 

e.g for today, 17th Nov 2019...

* ISO international standard: `2019-11-17` - I'd guess most users of Chocolatey are somewhat technical people, so this is typically the format we're used to.
* Something like `17th Nov 2019` or `2019-Nov-17`
* Or if you really want them in the zany American order 😜, please at least use `Nov/17/2019` so that it's clear what format is being used.

**Comment by designxsteph** [comment]
Hi @hi2u, thanks for bringing this up. A fix for this is the works and there is a related conversation about this over at https://github.com/chocolatey/chocolatey.org/issues/647.

Related to https://github.com/chocolatey/chocolatey.org/issues/647

**Comment by ferventcoder** [comment]
Duplicate of #647

**Comment by ferventcoder** [comment]
Wrote that in a specific manner so it would be identified as a duplicate. Closing as a duplicate now. 

@hi2u appreciate the suggestion! We typically close issues that are already identified so we don't have duplicates. We don't always expect you to find those as the search is sometimes finicky, so we'll find them for you and do the legwork here. However, we can't subscribe you to the conversation on the other ticket, so if you would like to know when that releases, head over to #647 and click subscribe on the notifications on the right side (or just add  a comment for something that may not have been brought up yet). This way you will know when we've addressed the issue and released the fixes.
